docs/UserGuide.md

```diff
@@ -162,6 +176,19 @@ Examples:
 * `edit 2 g/f`
 * `edit 3 g/F`
 
+__Optional Parameter 2: Date of Birth__
+
+Format: `edit INDEX [n/NAME] [d/DOB]`
+
+- Edits the date of birth of a person in the contact list. Date formats accepted are: d.M.yyyy, dd MMM yyyy, M d, yyyy
+- `INDEX must be positive integers (i.e 1,2,3…)`
+- `INDEX must be within the range of the contact list index (i.e from 1 to size of contact list).`
```

Comment:
Would be better to format only the INDEX and not the whole line! Also, to standardise with QR & since its singular, might want to use the term 'a' positive integer
```suggestion
- `INDEX` must be **a positive integer** (i.e 1,2,3…)
- `INDEX` must be within the range of the contact list index (i.e from 1 to size of contact list).
```

Reply:
mmmm ic!